src/views/HomeView.js

```diff
@@ -0,0 +1,179 @@
+import {Helper} from "../Helper";
+import {AbstractView} from "./AbstractView";
+
+/**
+ * Home View Class.
+ *
+ * Used to create a Home view.
+ *
+ * @class HomeView
+ * @extends AbstractView
+ */
+class HomeView extends AbstractView {
+  /**
+   * Domain of the view's entities.
+   * @type {string}
+   */
+  #domain = "camera";
```

Comment:
Why is the domain `camera`?

Reply:
This variable and its getter method are also probably a copy/paste leftover.
Will be removed.